go-controller/pkg/libovsdbops/acl.go

```diff
@@ -151,6 +150,11 @@ func BuildACL(name string, direction nbdb.ACLDirection, priority int, match stri
 		Meter:       realMeter,
 		ExternalIDs: externalIds,
 	}
+	if direction == nbdb.ACLDirectionFromLport {
+		acl.Options = map[string]string{"apply-after-lb": "true"}
+	}
```

Comment:
I think this should be set outside of libovsdbops, the relationship between `direction == nbdb.ACLDirectionFromLport` and `acl.Options = map[string]string{"apply-after-lb": "true"}` is a business logic that's out of context in a package that is strictly for performing liovsdb operations.

Reply:
@jcaamano does what I have now look better?